src/SeaNetMicron.cpp

```diff
@@ -53,35 +53,44 @@ void Micron::decodeSonarBeam(base::samples::SonarBeam &sonar_beam)
     }
 
     //copy data into SonarBeam
-    sonar_beam.time = base::Time::now();
-    sonar_beam.sampling_interval  = ((640.0*data.ad_interval)*1e-9);
-    sonar_beam.bearing = base::Angle::fromRad(-(data.bearing/6399.0*2.0*M_PI)+M_PI);
```

Comment:
@doudou and @saarnold,

I'm sure about that change (left and right limit) because it is related with this one. I don't know why the bearing is inverted in this part of source code and the SonarView plots inverted too. So if the bearing decreases, the legacy sonar viewer (SonarView) will plot the data from left to right.